lima.wdl

```diff
@@ -58,7 +58,6 @@ task Lima {
 
     command {
         set -e
-        mkdir -p "$(dirname ~{outputPrefix})"
```

Comment:
What is the reason for the changes here, going back to the glob structure?

Reply:
It's a lot simpler, and since we no longer support multiple input bam files will be no name collisions.

Reply:
But you do get these glob folders in the output now right? Or does that not happen when its just one file per glob?

Also, shouldn't the task check the possible output directory if that is set in the main pipeline?
Since you've remove `mkdir -p "$(dirname ~{outputPrefix})"` now?

Reply:
I've also removed the outputPrefix from the subreads-processing pipeline, so that is not a problem. I've been using these tasks for a few weeks already, so they all work.